pytransform3d/transform_manager/_transform_graph_base.pyi

```diff
@@ -16,22 +16,18 @@ class TransformGraphBase(abc.ABC):
     dist: np.ndarray
     predecessors: np.ndarray
     _cached_shortest_paths: Dict[Tuple[int, int], List[Hashable]]
+    _transforms = Dict[Tuple[Hashable, Hashable], Any]
```

Comment:
We can put here what we want, and the get_transforms should call the interface/protocol ... i.e. deal with numpy arrays or `TimeTransform` instances